site2/docs/helm-deploy.md

```diff
@@ -131,49 +128,49 @@ The Pulsar Helm chart is designed to enable controlled upgrades. So it can confi
 images:
   zookeeper:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   bookie:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   autorecovery:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   broker:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   proxy:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
     pullPolicy: IfNotPresent
   functions:
     repository: apachepulsar/pulsar-all
-    tag: 2.5.0
+    tag: @pulsar:version@
   prometheus:
     repository: prom/prometheus
-    tag: v1.6.3
+    tag: v2.17.2
     pullPolicy: IfNotPresent
   grafana:
     repository: streamnative/apache-pulsar-grafana-dashboard-k8s
-    tag: 0.0.4
+    tag: 0.0.16
     pullPolicy: IfNotPresent
```

Comment:
We no longer deploy these components this way. We can remove this section.

Configuration for prometheus and grafana is governed by https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack. It just needs to be in the `kube-prometheus-stack:` section of the values.yaml file.